### ISSUE TITLE
Controlling frame rate of an Animation?

### ISSUE THREAD

**Issue by thertzelle** [visgl/deck.gl#5231]
It it possible to control the frame rate of an animation transition? We have a few use cases where drawing at 60 fps for an animation is not needed and is too performance heavy for our application. I've done a little searching, but cannot find a direct way to do this, so I was looking for some suggestions?

Thank you.

**Comment by Pessimistress** [comment]
What kind of animation are you talking about? Attribute transition? Viewport transition? Data updates?

**Comment by thertzelle** [comment]
Thank you for your reply and questions. I'm hoping to animate one to six ScatterplotLayer circles to represent a real-time wave traveling from a point. Last night I was using an update trigger to change the radius (with a transition on the radius). Then I saw the notes in (https://deck.gl/docs/developer-guide/performance) but it wasn't exactly about animation, so I switched it to update the radius and the performance improved slightly, but my final temp solution is to extremely manually update the radius without animation (looks a bit stuttery compared to the layer transition animation. Any tips here? The radius changes perhaps every second due to the wave being re-calculated based on speed.
